src/mixins/metadata.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import { addMetadata } from 'docc-render/utils/metadata';
+
+export default {
+  methods: {
+    /**
+     * Extract and join all the values from key 'text' on every object inside an array of objects,
+     * if there is an inlineContent array, it will extract all its text recursively as well
+     * @param {Array} content - array of objects containing "text" key
+     * @returns {String}
+     */
+    extractText(content) {
```

Comment:
We should add some unit tests for this.

Reply:
https://github.com/apple/swift-docc-render/pull/46/commits/79661b0867292c50c3cd4a6b990f53ea1cfb4a48